tests/unit/test_system.py

```diff
@@ -162,11 +164,10 @@ def test_unattended_upgrades(self):
         issues = list(IssuesStore().load().values())[0]
         self.assertEqual([issue['desc'] for issue in issues], [msg])
 
+    @mock.patch('hotsos.core.plugins.system.system.SystemBase.'
+                'virtualisation_type', None)
```

Comment:
Should we also test kvm here as well?

Reply:
the check says ne 'lxc' so should be good, I used None in this test to make sure it would be a valid comparison.